src/index.js

```diff
@@ -139,7 +141,9 @@ SentrySourceMapPlugin.prototype.apply = function(compiler) {
                             files[fileName].existsAt,
                             this.options.publicPaths[fileName] || publicPath,
                             fileName,
-                        ),
+                        ).catch((error) => compilation.warnings.push(
+			    `\Failed to upload source maps for: ${fileName}`
```

Comment:
Why do we need the `\` at the start? Or is that a mistake?  Shouldn't that be `\n` ? 

Also, it might be useful to print the actual error that occurred.